src/serializers/index.ts

```diff
@@ -77,14 +80,26 @@ const res = (response: Response) =>
       }
     : response;
 
-export const createSerializers = (opts: SerializerOptions) => {
+const createErrSerializer =
+  (
+    serializer: (error: Error) => SerializedError,
+    opts: FormatterOptions,
+  ): SerializerFn =>
+  (error: unknown): unknown =>
+    trimmer({
```

Comment:
Is there any value in sharing a trimmer across the serializers? (I have no clue.)